### DISCUSSION TITLE
能否增加“”两步路户外助手“广告屏蔽规则，谢谢

### DISCUSSION THREAD

**Discussion by xueheyunnd** [gkd-kit/gkd#150] [Q&A]
目前玩户外人原来越多，两步路户外助手 app广告能否帮助去掉谢谢！毕竟玩户外用的人也比较多。 第一次见到有如此胸怀的软件gkd，方便自己，方便大家。

**Comment by lisonge** [comment]
https://github.com/gkd-kit/subscription/issues/new/choose
